api_tests/preprint_providers/views/test_preprint_provider_subjects_list.py

```diff
@@ -11,9 +12,9 @@ def create_subject_rules(self):
         '''
         Subject Hierarchy
         +-----------------------------+
-        |                    +-->E    |
-        |      +-------->B+--+        |
```

Comment:
Why was this hierarchy changed? Aren't we still supporting subjects with multiple parents?